packages/graphback-core/src/db/getPrimaryKey.ts

```diff
@@ -14,10 +14,10 @@ export function getPrimaryKey(graphqlType: GraphQLObjectType): GraphQLField<any,
   let primaryKey: GraphQLField<any, any>;
   let primariesCount = 0;
   for (const field of fields) {
-    const dbConfig: any = parseDbAnnotations(field);
+    const hasIdMarker = parseMarker("id",String(field.description));
```

Comment:
This should work too:

```suggestion
    const hasIdMarker = parseMetadata("id", field);
```